packages/components/fields/multiline-text-field/src/multiline-text-field.tsx

```diff
@@ -0,0 +1,241 @@
+import {
+  type ChangeEventHandler,
+  type FocusEventHandler,
+  type ReactElement,
+  type ReactNode,
+  isValidElement,
+  Component,
+} from 'react';
+import {
+  createSequentialId,
+  filterDataAttributes,
+  getFieldId,
+  warning,
+} from '@commercetools-uikit/utils';
+import Constraints from '@commercetools-uikit/constraints';
+import Spacings from '@commercetools-uikit/spacings';
+import FieldLabel from '@commercetools-uikit/field-label';
+import MultilineTextInput from '@commercetools-uikit/multiline-text-input';
+import FieldErrors from '@commercetools-uikit/field-errors';
+
+export type TFieldErrors = Record<string, boolean>;
+
+export type TMultiTextFieldProps = {
+  // FieldLabel
+  /**
+   * Used as HTML id property. An id is auto-generated when it is not specified.
+   */
+  id?: string;
+  /**
+   * Horizontal size limit of the input fields.
+   */
+  horizontalConstraint?:
+    | 6
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 'scale'
+    | 'auto';
+  /**
+   * Called with custom errors. This function can return a message which will be wrapped in an ErrorMessage. It can also return null to show no error.
+   * <br />
+   * Signature: `(key, error) => React.node`
+   */
+  renderError?: TErrorRenderer;
+  /**
+   * Indicates if the value is required. Shows an the "required asterisk" if so.
+   */
+  isRequired?: boolean;
+  /**
+   * Indicates whether the field was touched. Errors will only be shown when the field was touched.
+   */
+  touched?: boolean;
+
+  // TextInput
+  /**
+   * Used as HTML `autocomplete` property
+   */
+  autoComplete?: string;
+  /**
+   * Used as HTML name of the input component. property
+   */
+  name?: string;
+  /**
+   * Value of the input component.
+   */
+  value: number | string;
```

Comment:
No idea why we had it as string or number. I think it should just be `string`.

Reply:
Thank you.